docs/DeveloperGuide.md

```diff
@@ -736,74 +736,74 @@ testers are expected to do more *exploratory* testing.
 
 1. Adding a guest
 
-   1. Prerequisites: Add a guest use add command, only one adding guest
-      the name of the adding guest should not in the guest book
-      Every part of guest information should be valid
+   1. Prerequisites: Only one guest to be added; 
+   the name of the guest to be added should not be already in the guest book; 
```

Comment:
Should you rewrite this as "the name of the guest to be added should not exist in the guest book;"?